populate.py

```diff
@@ -1177,35 +199,35 @@
 
 
 def gen_sentence():
-    return " ".join(random.sample(lorems, 50))
+    return fake.text()
 
 
 def gen_name():
-    return random.choice(names)
+    return fake.name()
```

Comment:
Can you make this return a first name only inline with previous behavior?

Reply:
yes

Reply:
```
def gen_name():
    from random import shuffle, seed
    from faker.providers.person.en import Provider

    first_names = list(set(Provider.first_names))
    seed(1551)
    shuffle(first_names)
    
    return first_names[0:1000]
```

with this new script I can print a list of 1000 first names without repetition.